frontend/src/pages/Registration/Registration.tsx

```diff
@@ -2,14 +2,15 @@ import React, { FC, useState } from 'react'
 import FormInput from '../../components/FormInput'
 import FormButton from '../../components/FormButton/FormButton'
 import { Container, Logo } from '../Login/Login.style'
-import {
-  NavButton,
-  NavButtonText,
-} from '../Login/Login.style'
-import { StyleSheet, Text, View, Button } from 'react-native';
+import { Button } from 'react-native'
+import { useNavigation } from '@react-navigation/native'
+import { StackNavigationProp } from '@react-navigation/stack'
+import { RootStackParamList } from '../../../App'
 
+type registrationNavProp = StackNavigationProp<RootStackParamList, 'Register'>
 
 export const Registration: FC = () => {
+  const { navigate } = useNavigation<registrationNavProp>()
```

Comment:
I found a navigation hook that gets the function we want to call for us `@react-navigation/native`. There's another example in `Login.tsx`. Hooks are an essential piece of knowledge for functional component look here for more info: https://reactjs.org/docs/hooks-overview.html